docs/cookbook/full-wallet.md

```diff
@@ -0,0 +1,43 @@
+# Full Wallet Example
+
+This page will do a walk through of 3 examples to illustrate core wallet functionality including:
+
+- Generating seeds and keys
```

Comment:
You can replace with generating descriptors (which is what the library focuses on). Within that you'll need keys. Seed phrases are sort of orthogonal to all of this and can be omitted here IMO.